tests/unit/test_config_node.py

```diff
@@ -252,6 +253,19 @@ def test_identify_templating_section(node_, expect, raises):
             identify_templating_section(node)
 
 
+@pytest.mark.parametrize(
+    'input_, expect',
+    (
+        ([None], 'template variables'),
+        (['jinja2'], 'jinja2:suite.rc'),
```

Comment:
Maybe test this case too?
```suggestion
        (['jinja2'], 'jinja2:suite.rc'),
        (['jinja2:suite.rc'], 'jinja2:suite.rc'),
```